src/quill-cursors/cursor.ts

```diff
@@ -106,6 +132,28 @@ export default class Cursor {
     selections.forEach((selection: ClientRect) => this._addSelection(selection, container));
   }
 
+  private _setHoverState(): void {
+    this._caretEl.classList.add(Cursor.CONTAINER_NO_POINTER_CLASS);
+    document.addEventListener('mouseover', this._toggleOpenedCursor);
+  }
+
+  private _toggleOpenedCursor(e: MouseEvent): void {
+    const shouldShow = this.toggleNearCursor(e.clientX, e.clientY);
+    this._caretEl.classList.toggle(Cursor.CONTAINER_NO_POINTER_CLASS, shouldShow);
+    document.removeEventListener('mouseover', this._toggleOpenedCursor);
+    if (!shouldShow) {
+      this._showFlagTimeout = null;
+      return;
+    }
+
+    this._showFlagTimeout = setTimeout(this._setHoverState, 100);
+  }
+
+  private _updateCoordinates(): void {
+    const {x, y, width, height} = this._caretEl.getBoundingClientRect();
+    this.coordinates = {x, y, width, height};
```

Comment:
I think our maths would be more readable if we changed to:

```suggestion
    const {left, right, top, bottom} = this._caretEl.getBoundingClientRect();
    this.coordinates = {left, right, top, bottom};
```